src/interfaces.ts

```diff
@@ -330,18 +330,18 @@ export interface IConversationDetails {
     description?: string;
     roles: IConversationRoles;
     isPublic: boolean;
-    participants?: IConversationParticipant[];
 }
 
 /**
  * Conversation details interface
  */
 export interface IConversationDetails2 extends IConversationDetails {
```

Comment:
Ok, let's try and avoid this naming style for anything in future ;) 

Approving this PR though.